pyiron_base/state/install.py

```diff
@@ -94,8 +94,12 @@ def _write_config_file(
             os.makedirs(project_path)
 
 
-def install_dialog():
-    user_input = None
+def install_dialog(input_by_user=True):
+    if input_by_user:
+        user_input = None
+    else:
+        user_input = "yes"
```

Comment:
```suggestion
def install_dialog(silently=False):
    if not silently:
        user_input = None
    else:
        user_input = "yes"
```

I think silently is slightly more consistent with e.g. `Project.remove_jobs()`.